.travis.yml

```
@@ -16,6 +23,9 @@ matrix:
   - name: Code Style Check
     php: 7.3
     env: CHECK_CS=1
+  - name: "Workflow Integration tests"
```

```suggestion
  - name: "User Integration tests"
```
Or something similar 😉